File: frontend/src/app/car.ts

export class Car {
  id: number;
  make: string;
  model: string;
  numberplate: string;

  constructor(id: number, make: string, model: string, numberplate: string) {
    this.id = id;
    this.make = make;
    this.model = model;
    this.numberplate = numberplate;
  }
}

File: src/main/java/com/home/user/model/Car.java
package com.home.user.model;

import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;
import lombok.RequiredArgsConstructor;

@Data
@AllArgsConstructor
@NoArgsConstructor
public class Car {
    int id;
    String make;
    String model;
    String numberplate;

}
